### ISSUE TITLE
[MD] Bad Apple MD demo 

### ISSUE THREAD

**Issue by freq-mod** [higan-emu/ares#831]
**Describe the issue**
[Bad Apple MD](https://github.com/NatsumiFox/Bad-Apple-MD) is a demo that abuses Mega Drive bank switching to produce 26 MB demo featuring 60 fps full motion video and crystal clear sampled PCM playback. The first works. The second... it's a completely garbled mess. Towards the end of a demo (around 3 minutes in), the graphics go away as well, with some light-colored stripes being displayed.

**Rom Information**

```
file: badapple.md
sha256: 16482aa4de8fc81e00ddd186f6716f35c7c9574589cc5d0a66bed89c2a43b58c
```

**Comment by TascoDLX** [comment]
https://github.com/ares-emulator/ares/blob/ed43ab72b225ce0746c7eb8f704ea41891a3b4e0/ares/md/cartridge/board/standard.cpp#L62
^ offset should be n25 to support the full 32MB rom range. I think that's the only problem here, if anyone wants to verify.

**Comment by invertego** [comment]
@TascoDLX fixed this with #852
